gapic/generator/options.py

```diff
@@ -92,12 +92,19 @@ def build(cls, opt_string: str) -> 'Options':
                 os.path.join(os.path.dirname(__file__), '..', 'templates'),
             )
 
+        retry_cfg = None
+        retry_paths = opts.pop(cls.RETRY_OPT, None)
```

Comment:
Can we just make `cls.RETRY_OPT` point to a single file?

Reply:
The way that the option string is parsed makes it tricky to do that. It could be done, but it would require special casing `cls.RETRY_OPT` or making a general solution that other options do not pick by default.

Reply:
That sounds reasonable.

Reply:
I would lean towards keeping the parsing simple and general. If there are ever more options that should only take a single value, generalizing the parsing can be revisited.